packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -1268,6 +1277,78 @@ const en = {
       title: 'Are you sure you want to exit?',
     },
   },
+  toggle: {
+    on: 'ON',
+    off: 'OFF',
+  },
+  settings: {
+    tabs: {
+      general: {
+        title: 'General',
+        item: {
+          syncMobile: 'Sync cySync Mobile App',
+          syncMobileDesc:
+            'Sync your Accounts with cySync Mobile App through QR code',
```

Comment:
I think this will be better
```suggestion
          syncMobile: {
              title: 'Sync cySync Mobile App',
              description:
                'Sync your Accounts with cySync Mobile App through QR code',
          },
```

Reply:
suggestions applied